src/actions/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { createAction } from "redux-act";
+import { Product } from "../pages/Products/util";
+import { Category } from "../pages/Category/util";
+
+export const addItem = createAction<Product>("add item to the cart");
+export const fetchProductsResult = createAction<Product[]>(
+  "FETCH_ALL_PRODUCTS_ON_SUCCESS"
```

Comment:
"fetch products success", it's easier to read
```suggestion
  "fetch products success"
```